articles/enroll-hosts.md

```diff
@@ -311,6 +312,47 @@ If this setting is not used, you will need to configure client TLS certificates
 
 To use mTLS use the [AutoSelectCertificateForUrls policy](https://chromeenterprise.google/policies/?policy=AutoSelectCertificateForUrls) to point Chrome to your client certificates
 
+### Using host identity certificates
+
+`Applies only to Fleet Premium`
+
+Host identity certificates allow Fleet's agent (fleetd) to use hardware-backed client certificates for authentication to the Fleet server. This feature uses the host's TPM (Trusted Platform Module) to generate and store cryptographic keys, providing strong hardware-based authentication.
+
+This provides a level of security similar to [mTLS](#using-mtls), but the certificate is hardware-backed and managed by the TPM rather than being stored as a file on disk. The TPM ensures the private key cannot be extracted or copied, providing stronger security guarantees.
+
+#### Requirements
+
+- Linux hosts only
+- TPM 2.0 hardware (or vTPM for VMs)
+- Linux kernel 4.12 or later
+- Fleet Premium license
+
+#### Generating fleetd with host identity certificates
+
+When generating fleetd with host identity certificates, use the `--fleet-managed-client-certificate` flag:
```

Comment:
```suggestion
To use host identity certificates, generate fleetd with the `--fleet-managed-client-certificate` flag:
```

Cleaned up the language here. As a user, I want to know how to use identity certs.